src/DeviceRenderer.js

```diff
@@ -478,6 +485,17 @@ module.exports = class DeviceRenderer {
                 this.gamepadManager.addGamepadCallbacks();
             }
 
+            // in case this is a re-start, we want to add the callbacks again depending on the state
+            if (this.store.get('KEYBOARD_EVENTS_ENABLED')) {
+                this.keyboardEvents.addKeyboardCallbacks();
+            }
+            if (this.store.get('MOUSE_EVENTS_ENABLED')) {
+                this.mouseEvents.addMouseCallbacks();
+            }
+            if (!this.store.get('KEYBOARD_EVENTS_ENABLED') && !this.store.get('MOUSE_EVENTS_ENABLED')) {
+                this.keyboardMapping.addKeyboardCallbacks();
+            }
```

Comment:
These getters don't actually work for now, this is for discussion purposes. A working diff would look like this, but I don't think these state properties are supposed to be public-facing?
```suggestion
            // in case this is a re-start, we want to add the callbacks again depending on the state
            if (this.store.state.isKeyboardEventsEnabled) {
                this.keyboardEvents.addKeyboardCallbacks();
            }
            if (this.store.state.isMouseEventsEnabled) {
                this.mouseEvents.addMouseCallbacks();
            }
            if (!this.store.state.isKeyboardEventsEnabled && !this.store.state.isMouseEventsEnabled) {
                this.keyboardMapping.addKeyboardCallbacks();
            }
```

Reply:
this.store.state is the right way to get values from the store